3_navigation.ipynb

```diff
@@ -83,7 +155,29 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Now, we specify the goal of autonomous movement from RViz. Click \"2D Nav Goal\" on the toolbar at the top of RViz and click an unobstructed location in the room:\n",
+    "\n",
+    "![title](./imgs/3_2d_nav_goal_button.png)\n",
+    "![title](./imgs/3_2d_nav_goal.png)\n",
+    "\n",
+    "HSR starts autonomous movement. On the RViz screen or the simulator screen, check that HSR moves to the specified position while avoiding obstacles.\n",
```

Comment:
"The HSR starts autonomous movement. On the RViz screen or the simulator screen, check that the HSR moves to the specified position while avoiding obstacles.\n",